lib/nostrum/struct/message.ex

```diff
@@ -100,6 +102,11 @@ defmodule Nostrum.Struct.Message do
   """
   @type application :: Application.t() | nil
 
+  @typedoc """
+  Partial Guild Member object received with Message Create event if message came from a guild channel.
```

Comment:
Like before, I think a `the` would make it read better before "Message Create event"